code/__HELPERS/type2type.dm

```diff
@@ -385,9 +385,9 @@
 		switch(child)
 			if(/datum)
 				return null
-			if(/obj || /mob)
+			if(/obj, /mob)
```

Comment:
Seems like only components use this for their logic. Don't think stuff will break or change due to how our components are set up though.